packages/itwinui-css/src/tree/tree.scss

```diff
@@ -34,7 +34,7 @@ $iui-expander-button-width: calc(var(--iui-component-height-small) + 2px);
   min-inline-size: max-content;
 
   &:focus-visible {
-    outline: none;
+    outline: none !important;
```

Comment:
Is this the only place we needed to add `!important`? I would have expected that the unlayered high-specificity `outline` declaration would override everything.

Reply:
They're not that bad, just some areas where the 2px outline gets clipped so it only shows 1px:
<img width="611" height="62" alt="Screenshot 2025-10-06 at 4 54 48 PM" src="https://github.com/user-attachments/assets/8bb0876f-3d13-4d35-b0a1-3946aacbdb37" />
<img width="329" height="213" alt="Screenshot 2025-10-06 at 4 55 39 PM" src="https://github.com/user-attachments/assets/643fb104-4db8-49ae-8c93-9d602450f826" />
<img width="434" height="162" alt="Screenshot 2025-10-06 at 4 56 39 PM" src="https://github.com/user-attachments/assets/5a354476-1ab2-4f4f-988e-9b10384b31df" />
<img width="781" height="134" alt="Screenshot 2025-10-06 at 4 57 15 PM" src="https://github.com/user-attachments/assets/e5a4094a-02cf-4da8-8b10-182475735260" />